Guidelet/GuideletLib/Guidelet.py

```diff
@@ -221,7 +221,7 @@ def setupAdditionalPanel(self):
     pass
 
   def registerLayout(self, layoutName, layoutId, layoutXmlDescription, layoutSelectCallback=None):
-    if (type(layoutName) != str and type(layoutName) != unicode) or len(layoutName) < 1:
+    if (type(layoutName) != str) or len(layoutName) < 1:
```

Comment:
This code has to work in the Stable version of Slicer as well. type(layoutName) will be 'unicode' in Slicer 4.10.2.

If there isn't a way to do type checking that works on both platforms, you can check the Slicer version before determining the type using:
```
if slicer.app.majorVersion >= 5 or (slicer.app.majorVersion >= 4 and slicer.app.minorVersion >= 11):
```